### ISSUE TITLE
Error while typing alt text in an image

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/hhmi#309]
> **Migrated from GitLab**
> **Original Author:** Vignesh (`@vignesh03`)
> **Original Link:** https://gitlab.coko.foundation/coko-org/products/hhmi/hhmi/-/issues/172

While typing the `Alt text` for an image, Cypress detects the error `Cannot read properties of undefined (reading 'startsWith')`.

This happens as the image disappears when alt text is being typed by Cypress.

## Steps to reproduce
![image](/uploads/ae60a5bfe83941ed8c36f8c0f6234c03/image.png)

1. switch to `cypress-tests` branch.
2. run `yarn run cypress open`.
3. choose `E2E testing` and run `Test02_questions`
4. Notice that the test `check alternative text for empty questions` fails when typing the alt text

## Environment

[Cypress(electron browser),Mac]
## Scheduling

- [ ] This issue is blocking current user testing on staging stie
- [ ] This issue occurs on the production site

**Comment by yannisbarlas** [comment]
> **Migrated Comment**
> **Author:** Vignesh (`@vignesh03`)
> **Date:** 2023-11-15T06:27:42.517Z

`@DioneMentis`  `@Natasha` , I enabled the test block that was throwing this error and it doesn't throw any error now. I guess the recent wax upgrades fixed it.
